contactsummary/js/src/contact-summary/controllers/keyDetails.js

```diff
@@ -42,11 +42,38 @@ define([
     // Private Members //
     /////////////////////
 
+    function formatDuration(duration) {
+        var texts = [],
+            days = duration.days(),
+            months = duration.months(),
+            years = duration.years();
+
+        if (years > 0) {
+            texts.push(years + " years");
```

Comment:
Aren't these `years`, `months`, `days` more appropriate for the view layer? Shouldn't you use a `ng-if` directive on the template?